config/routes.php

```diff
@@ -6,5 +6,45 @@
  * @var array $params
  */
 
+use Psr\Http\Message\ResponseFactoryInterface;
+use Yiisoft\DataResponse\Middleware\FormatDataResponseAsJson;
+use Yiisoft\Router\Group;
+use Yiisoft\Router\Route;
+use Yiisoft\Validator\ValidatorInterface;
+use Yiisoft\Yii\Debug\Api\Middleware\ResponseDataWrapper;
+use Yiisoft\Yii\Gii\Controller\DefaultController;
+use Yiisoft\Yii\Middleware\IpFilter;
+
+if (!(bool)($params['yiisoft/yii-enabled']['enabled'] ?? false)) {
```

Comment:
Seems like there is a typo in `yii-enabled`?

Reply:
Fixed